### ISSUE TITLE
[advanced search] openedFacetList is not working

### ISSUE THREAD

**Issue by JoanaDalmeida** [KleeGroup/focus-components#662]
# Actual behaviour

I set openedFacetList props to my AdvanceSearch component, but after render all the facets are expanded.

# Expected behaviour

Only openedFacetList must be expanded.

# Versions 

Focus-core : 0.11.3
Focus-components : 0.7.4

**Comment by sgoretti** [comment]
:+1: 
We overloaded the components in my project to have it work:
(I'll be scolded by the Focus team, but here it is:)

First, the Advances search overload.

```
// mixins.
let advancedSearchMixin = FocusComponents.page.search.advancedSearch.mixin;

// components
let FacetBox = require('./facet-box').component;
let builder = FocusCore.component.builder;
let type = FocusCore.component.types;

/**
 * facet box overloaded to force active when selecting scope.
 * @type {Object}
 */
 let advancedSearch = {
	mixins: [_.omit(advancedSearchMixin, '_renderFacetBox')],
	_renderFacetBox() {
		const {facets, selectedFacets} = this.state;
        const {facetConfig, scopesConfig, openedFacetList} = this.props;
        return (
            <FacetBox
                action={this._action}
                facetConfig={facetConfig}
            	openedFacetList={openedFacetList}
                facets={facets}
                scopesConfig={scopesConfig}
                selectedFacets={selectedFacets}
                />
        );
    }
};
 
 module.exports = builder(advancedSearch);
```

Since I unfortunately cant call the advanced-search FacetBox (in focus-compnents/src/page/search/advanced/search/facet-box.js, masked by the index.js of the advanced search), I have to copy it entirely in my app (and I don't like it).

Come to think of it, I'll do a pull request with it

**Comment by Bernardstanislas** [comment]
@sgoretti fixed it in #665
